runtime/doc/news.txt

```diff
@@ -192,6 +192,8 @@ These existing features changed their behavior.
 • 'spellfile' location defaults to `stdpath("data").."/site/spell/"` instead of
   the first writable directory in 'runtimepath'.
 • |vim.version.range()| doesn't exclude `to` if it is equal to `from`.
+• Multigrid UIs can optionally pass 0 as the mouse grid to let Nvim automatically
```

Comment:
Let's move this up to news-features, either in UI or API section.

Reply:
Moved to UI